spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -16,7 +16,8 @@
       cc: ['emily@example.com'],
       from: 'Joe User <joeuser@example.com>',
       subject: 'Re: [ThisApp] That thing',
-      text: /Dear bob/
+      text: /Dear bob/,
+      headers: { Subject: 'Re: [ThisApp] That thing', Cc: 'emily@example.com' }
```

Comment:
I used single quotes for consistency since that is how the rest of the spec file is written. I can change the whole file to double quotes if you want